File: src/test/java/com/fractalwrench/bpp/programs/BppEchoArgs.java
package com.fractalwrench.bpp.programs;

import org.junit.Test;

public class BppEchoArgs extends BppTestCase {

    @Test
    public void checkEchoArgs() throws Exception {
        runBppSample("EchoArgs", new String[]{"Echo!"}, "Echo!\n");
    }

}

File: src/test/java/com/fractalwrench/bpp/programs/BppEchoArgs.kt
package com.fractalwrench.bpp.programs

import org.junit.Test

class BppEchoArgs : BppTestCase() {

    @Test
    @Throws(Exception::class)
    fun checkEchoArgs() {
        runBppSample("EchoArgs", arrayOf("Echo!"), "Echo!\n")
    }

}
